src/main/scala/no/ndla/articleapi/service/search/ArticleSearchService.scala

```diff
@@ -102,6 +104,12 @@ trait ArticleSearchService {
         .setParameter(Parameters.SIZE, numResults)
         .setParameter("from", startAt)
 
+      val requestedResultWindow = pageSize * page
+      if (requestedResultWindow > ArticleApiProperties.ElasticSearchIndexMaxResultWindow) {
+        logger.error(s"Max supported results are ${ArticleApiProperties.ElasticSearchIndexMaxResultWindow}, user requested ${requestedResultWindow}")
```

Comment:
Burde denne kanskje være `logger.info` siden feilen blir håndtert?

Reply:
Tja, kanskje warning? Har egentlig ikke et veldig bestemt forhold til hvilke log-nivå det bør være enda. Tenker bare om jeg holdt på å debugge en krasj eller noe ville nok det være litt misvisende at denne logges som en error siden den, som du sier, håndteres.

Reply:
Gikk for en info da jeg føler det kan bli litt dumt dersom brukere kan spamme loggen med warnings som egentlig ikke betyr at noe er galt.